### PR TITLE
Remove confusing 'inactive' caption (fixes #11823)

### LINKED ISSUE #11823: With WebExtension in Firefox Nightly "customize mode", HTTPS Everywhere icon's title says "(inactive)" which is misleading
STR:
 1. Start Firefox Nightly
 2. Install latest HTTPS Everywhere WebExtension from https://www.eff.org/files/https-everywhere-test/index.html
(I'm using https-everywhere-2017.7.21.1338-eff.xpi )
 3. Click the Firefox hamburger menu (top right of UI) and choose "Customize..."
 4. Drag the HTTPS Everywhere icon from your toolbar onto the overflow menu (directly below it)

ACTUAL RESULTS:
The HTTPS Everywhere entry in the overflow menu has this text "HTTPS Everywhere (inactive)" which gives the impression that the extension is inactive/disabled somehow. Screenshot:
Screenshot

EXPECTED RESULTS:
The entry should just say "HTTPS Everywhere", probably.

### PR DESCRIPTION
**Pull request by koops76**
@Hainish Your opinion? Detecting Customize tab is not very easy since its URL is `about:blank`.

### CODE REVIEW

**Comment by koops76** [comment]
@Hainish Different icon for active/inactive is superfluous since now we have a counter. I guess removing it is a good idea, especially since it may confuse the user.

**Comment by koops76** [comment]
@Hainish Review please so it will be included in the release.

File: chromium/background.js [hunk 1]
```diff
@@ -157,42 +157,42 @@ function getActiveRulesetCount(id) {
...
  * Set the icon color correctly
- * inactive: extension is enabled, but no rules were triggered on this page.
+ * active: extension is enabled.
  * blocking: extension is in "block all HTTP requests" mode.
- * active: extension is enabled and rewrote URLs on this page.
  * disabled: extension is disabled from the popup menu.
  */
-var updateState = function() {
-  if (!chrome.tabs) {
-    return;
```
Comment (Hainish) [inline chromium/background.js]: We need these lines for Fennec

**Comment by Hainish** [comment]
Updated the above comment. s/disabled/inactive/g

File: chromium/background.js (hunk 1 above)
Comment (koops76) [reply chromium/background.js]: @Hainish Done, anything else?

**Comment by Hainish** [comment]
Testing now...

**Comment by Hainish** [comment]
lgtm, merging after CI runs its course

**Pull request merged**